test/Microsoft.AspNet.Razor.Test/TestFiles/CodeGenerator/CS/Source/EmptyAttributeTagHelpers.cshtml

```diff
@@ -0,0 +1,8 @@
+﻿@addtaghelper "something"
+
+<div>
+    <input type= checked=""class="" />
+    <p age=''>
+        <input type=""checked= class="" />
```

Comment:
It's not an error according to the Razor parser. it's essentially parsing the attribute as if it was going to be represented like: `<input checked=true />` minus the value.